### ISSUE TITLE
File upload not working for large files

### ISSUE THREAD

**Issue by chmeyers** [googledatalab/datalab#1324]
Uploading a large file doesn't seem to work.  User was able to upload one that was 151KB, but not one that was 4.8MB.

<img width="740" alt="screen shot 2017-04-19 at 4 45 59 pm" src="https://cloud.githubusercontent.com/assets/11337551/25206563/913bfeea-251d-11e7-8802-a5dc91b2ff6a.png">

**Comment by chmeyers** [comment]
Not sure if this is specific to running on cloud through the SSH tunnel or whether it will also happen locally.

**Comment by ttoleung** [comment]
I am having the same issue, is this resolved ?

**Comment by yebrahim** [comment]
We inherit this from Jupyter unfortunately, there isn't much we can do about it without updating the Jupyter version. See: https://github.com/jupyter/notebook/issues/650. It was fixed in 5.0.0: https://github.com/jupyter/notebook/pull/2162.

**Comment by yebrahim** [comment]
I'll keep this open as an incentive to push us to notebook==5.0.0.

**Comment by MaxGhenis** [comment]
I just hit this with a 1.8MB notebook.
![screenshot 2018-04-13 at 23 35 25](https://user-images.githubusercontent.com/6076111/38765288-8dd9a8c6-3f73-11e8-92c2-b5b5ccbea591.png)

Does copy/paste multiple cells from Jupyter notebooks work as a workaround? I'm not on a computer with that right now, but if not this will unfortunately be a deal-breaker for me.

**Comment by MaxGhenis** [comment]
> Does copy/paste multiple cells from Jupyter notebooks work as a workaround?

No, I filed https://github.com/googledatalab/datalab/issues/1992.

**Comment by netskink** [comment]
I guess I have a similar problem. I can not save my notebook after I added some graphs.  How did you get the size of the file?  I am still new to datalab and googlecloud.

**Comment by elsaad1808** [comment]
Any update on this case? I can't upload notebook with ~2MB size

**Comment by omar16100** [comment]
suffering big time on datalab, tried a workaround with GCS, doesn't work

**Comment by Deepakchawla** [comment]
Getting the same issue
